apps/openmw/mwrender/vismask.hpp

```diff
@@ -51,7 +51,9 @@ namespace MWRender
         Mask_PreCompile = (1<<16),
 
         // Set on a camera's cull mask to enable the LightManager
-        Mask_Lighting = (1<<17)
+        Mask_Lighting = (1<<17),
+
+        Mask_CastsShadows = (1<<18)
```

Comment:
This is an unused flag that I forgot to remove.

Reply:
I can tidy that up.